src/lib/statistics/chunk_statistics/histograms/histogram_utils.hpp

```diff
@@ -0,0 +1,104 @@
+#pragma once
+
```

Comment:
lot's of pretty generic function names in the global namespace. previous_value/next_value are probably okay, but starting from `base_value_for_prefix_length` it get's confusing as there is no context for the function name (what is a base value? what is a prefix?). Either put them in `namespace histogram` or prefix them all with `hist[ogram]_*`

Reply:
Used the namespace.